test/test_api.py

```diff
@@ -728,6 +734,61 @@ def test_update_access_control(self, mocker):
         self.assertIn("access_control", params)
         self.assertEqual(exp_acl, params["access_control"])
 
+    @unittest.skipUnless(cloudinary.config().api_secret, "requires api_key/api_secret")
+    def test_verify_api_response_signature(self):
```

Comment:
move tests to separate file as well, and try to align input and expected values with PHP SDK.